docs/python/debugging.md

```diff
@@ -170,11 +170,10 @@ Remote debugging allows you to step through a program locally within VS Code whi
 
 Windows:
 1. Enable ssh port forwarding on the remote computer using sshd_config or similar.
-2. Establish a Putty SSH tunnel:
-    <ol>
-    <li><a href="http://realprogrammers.com/how_to/set_up_an_ssh_tunnel_with_putty.html">http://realprogrammers.com/how_to/set_up_an_ssh_tunnel_with_putty.html</a> (until Open the session section).</li>
-    <li>On the Tunnels screen, using a local mode, source port (which is the port which will be the entry point on the local computer) can be different from the destination port (the end point on the server).</li>
-    <li>Destination address should be the localhost or 127.0.0.1 address (which is the address that the remote ssh server will use to establish the tunnel).</li></ol>
+2. Establish a PuTTY SSH tunnel:
+    1. http://realprogrammers.com/how_to/set_up_an_ssh_tunnel_with_putty.html (until "Open the session" section).
```

Comment:
Add the link back.

Reply:
Markdown auto-links URLs. Does the VS Code site not follow that?

Reply:
I didn't see it doing that on the local server, which is why I had the link there. @gregvanl do you know for certain?